generator/src/main/groovy/generator/SiteGenerator.groovy

```diff
@@ -287,7 +287,7 @@ class SiteGenerator {
 
     @CompileDynamic
     private void renderBlogFeed(Map<String, Document> blogList, String baseDir) {
-        def sorted = blogList.sort { e -> e.value.revisionInfo.date }
+        def sorted = blogList.sort { e -> e.value.revisionInfo.date.reverse() }
```

Comment:
Instead of using reverse (which should be outside the closure), this can easily be done using the other variant of sort:

```
def sorted = blogList.sort { a, b -> 
   b.value.revisionInfo.date <=> a.value.revisionInfo.date 
}
```

Reply:
👍 Yes, that's what I ended up going with.

Reply:
reverse() isn't currently available for Map's.